samples/create_event.cc

```diff
@@ -1,8 +1,127 @@
 #include "proto/spec.pb.h"
+#include <fstream>
 #include <iostream>
-int main() {
+
+void GetUserInput(std::string prompt, std::string& res) {
+    std::cout << prompt << ": ";
+    getline(std::cin, res);
+}
+
+void OutputToInterface(std::string content, int type) {
```

Comment:
What's the purpose of the `type`? I think we can pick a little more descriptive variable name here.

Reply:
changed to a boolean value of isError to be more descriptive